enhancements/installer/component-selection.md

```diff
@@ -92,235 +97,185 @@ an install experience that disables components that aren't needed for my use cas
-
-2) Defining an [install config api](https://github.com/openshift/installer/blob/790048166067273d34f76bea4220fa395b1cce1b/pkg/types/installconfig.go#L70) field whereby the user can opt out of specific capabilities.
-
-Example 1: include all capabilities by default, exclude some specific capabilities
-```yaml
-InstallConfig.capabilities
-  inclusionDefault: Include
-  exclude:
-  - console
-  - samples
-```
+Capabilities must be [registered in the API][capability-registry].
 
-Example 2: exclude all capabilities by default, include some specific capabilities.
-```yaml
-InstallConfig.capabilities
-  inclusionDefault: Exclude
-  include:
-  - registry
-```
+* 4.11
+  * baremetal
+  * marketplace
+  * openshift-samples
+* Maybe later:
+  * console
+  * imageregistry
+  * csi-*
+  * insights
+  * monitoring
+  * ???
 
+#### Capability sets
 
-To determine which resources ultimately get installed for a given cluster:
+Capability sets must be [registered in the API][capability-set-registry].
 
-1) start with all the resources in the payload
-2) remove any resources that don't match the configured [cluster profile](https://github.com/openshift/enhancements/blob/7da960b98b2835022823f569efa1e18e9e410696/enhancements/update/cluster-profiles.md)
+* `None`: an empty set enabling no optional capabilities.
+* `v4.11`: the recommended set of optional capabilities to enable for the 4.11 version of OpenShift.
+  This list will remain the same no matter which version of OpenShift is installed.
+* `vCurrent`: the recommended set of optional capabilities to enable for the cluster current version of OpenShift.
+  For a 4.11 cluster, this matches `v4.11`.  We may add or remove capabilities from this set in 4.12 clusters.
 
-From the remaining set of resources:
-3) if any resource is part of an explicitly excluded capability, remove it
-4) if any resource is part of an explicitly included capabilitity, keep it
-5) if the inclusionDefault is Exclude, remove all remaining resources(i.e. not covered by (4)) that are part of a capability
+Admins who want to pre-emptively exclude future capabilities can use `None` or `v4.11` or another `baselineCapabilitySet` which will not evolve with time.
 
+We may decide to add additional sets like `v4.12` or `myGreatUseCase` in the future, but do not commit to doing so.
 
-The `inclusionDefault` field (carried through to the CVO) allows an Admin to choose what behavior they want for new
-capabilities that might come as part of a future upgrade (either have the new capability installed during
-the upgrade, or exclude it).  The `inclusionDefault` field itself will default to `Include`.
+#### Manifest annotations
 
-Specifying the same capability as an `include` and an `exclude` is an error condition.
+Formalizing a concept of a "capability" annotation which allows a given resource to be excluded based
+on installer input. For example the console related resources could be annotated as
```

Comment:
This gives an impression that marking  a manifest part of a capability will make installer exclude it. Which is not true.

Reply:
how does it give that impression?  it says it allows the resource to be excluded via (config) input to the installer.

it doesn't say the resource will be excluded, and it doesn't say the installer will exclude it.

Reply:
Here is my suggestion `Formalizing a concept of a "capability" annotation  gives the ability to exclude the given resource based on installer input`

Reply:
Ben and I are fine either way, so going with Lala's wording in 7907bdc8 -> 7b7d1c9a.